src/main/kotlin/com/ampnet/web3provider/enums/RedisEntity.kt

```diff
@@ -3,5 +3,7 @@ package com.ampnet.web3provider.enums
 @Suppress("MagicNumber")
 enum class RedisEntity(val methodName: String, val ttlInSec: Long) {
     BALANCE("eth_getBalance", 5),
-    CODE("eth_getCode", 5)
+    CODE("eth_getCode", 5),
```

Comment:
Are durations ok here? You are caching this one for only 5 seconds, while `CHAIN_ID` gets cached for 30 days. Should it be like that?

Reply:
The exact requirements aren't agreed upon. 30 days for chainId was requested by @mightymatth, we can change the others later on.

Reply:
Ok, good to know. Maybe it would be good to put some TODOs here until we know exactly how long each entry should be stored for.

Reply:
Sure, I''l put a TODO